packages/polaris-viz/src/utilities/isComparisonSeriesHidden.ts

```diff
@@ -0,0 +1,24 @@
+import type {ComparisonSeriesIndex} from '@shopify/polaris-viz-core/src/types';
+
+export function isComparisonSeriesHidden({
```

Comment:
nit: maybe `hasHiddenComparisonSeries` would be clearer, to describe the `if (comparisonSeries == null) return false` case